packages/fxa-content-server/app/tests/spec/views/settings/delete_account.js

```diff
@@ -398,20 +406,8 @@ describe('views/settings/delete_account', function() {
         );
       });
 
-      it('renders only `active: true` subscriptions', () => {
-        assert.lengthOf(view.$('.delete-account-product-subscription'), 1);
-      });
-
-      it('renders subscription title attributes', () => {
```

Comment:
We had a couple of duplicate tests here that I removed, we're still testing subscription title attributes on line 377 and testing for active subscriptions on line 373.